src/sentry/utils/db.py

```diff
@@ -43,20 +43,20 @@ def has_charts(db):
     return True
 
 
-def attach_foreignkey(objects, field, related=[], database=None):
+def attach_foreignkey(objects, field):
```

Comment:
Is it ok to change the function signature here?

Reply:
Yes. This is only being used in one place and there were no parameters used there.

Reply:
:+1: